pkg/apis/servicecatalog/types.go

```diff
@@ -187,9 +198,11 @@ type Instance struct {
 // InstanceSpec represents a description of an Instance.
 type InstanceSpec struct {
 	// ServiceClassName is the reference to the ServiceClass this is an
-	// instance of.  Immutable.
+	// instance of.
+	// Immutable.
```

Comment:
Put an extra empty comment line in here if you want immutable to be drawn out and remain on it's own line. I know plenty of formatters that will squash it back up above.